R/workspace.R

```diff
@@ -157,11 +157,13 @@ get_workspace <- function(name, subscription_id = NULL, resource_group = NULL) {
 #' @param path A string of the path to the config file or starting directory
 #' for search. The parameter defaults to starting the search in the current
 #' directory.
+#' @param file_name A string that will override the config file name to
+#' search for when path is a directory path.
 #' @return The `Workspace` object.
 #' @export
 #' @md
-load_workspace_from_config <- function(path = NULL) {
-  azureml$core$workspace$Workspace$from_config(path)
+load_workspace_from_config <- function(path = NULL, file_name = NULL) {
+    azureml$core$workspace$Workspace$from_config(path, NULL, NULL, file_name)
```

Comment:
why not file_name = file_name so you don't need two NULLs.

Reply:
This is on purpose because in the Python SDK, the param is "_file_name", but R doesn't allow variables to begin with an underscore, so there's no way to use keyword arguments to pass in the file name.

Reply:
You can do this:
`azureml$core$workspace$Workspace$from_config(path="d:/code/.azureml", "_file_name"="config.json")`

Reply:
thanks!